tests/cases/setup_dbt7/vars.json

```diff
@@ -3,5 +3,5 @@
   "pg_data": "/opt/pgdata",
   "pg_wal": "/opt/pgwal",
   "limit_file_path": "/etc/security/limits.conf",
```

Comment:
is this good to remove now that core dump is handled differently?

Reply:
yes, will remove